public/app/core/utils/explore.test.ts

```diff
@@ -187,15 +187,15 @@ describe('hasNonEmptyQuery', () => {
 });
 
 describe('getTimeRange', () => {
-  describe('should flip from and to when from is after to', () => {
+  describe('should not flip from and to when from is after to', () => {
```

Comment:
I keep the test to test if we don't swap time ranges just in case we recover the swap behaviour, to make us check if we do the recovering purposefully.
I can delete the code if it is preferable.